js/src/dag.js

```diff
@@ -239,10 +246,16 @@ module.exports = (common) => {
           })
         })
 
-        it('dag-pb local scope', (done) => {
+        it('dag-pb local scope', function (done) {
+          // TODO vmx 2018-02-22: Currently not supported in go-ipfs, it might
+          // be possible once https://github.com/ipfs/go-ipfs/issues/4728 is
+          // done
+          if (withGo) {
+            this.skip()
+          }
           ipfs.dag.get(cidPb, 'Data', (err, result) => {
             expect(err).to.not.exist()
-            expect(result.value).to.eql(Buffer.from('I am inside a Protobuf'))
+            expect(result.value.data).to.eql(Buffer.from('I am inside a Protobuf'))
```

Comment:
Is this correct? Looks like `.data` should not be necessary.

Reply:
Good catch. This time I actually ran the tests on js-ipfs.